L/libjulia/common.jl

```diff
@@ -75,7 +77,11 @@ function build_julia(ARGS, version)
     override OS=Linux
     EOM
 
-    LLVM_CXXFLAGS="-I${prefix}/include -std=c++11 -fno-exceptions -fno-rtti -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS"
+    if [[ "${version}" == 1.6.* ]]; then
```

Comment:
I would switch the check around and do `if [[ "${version}" == 1.[0-5].* ]]; then` to make this future proof